### PR TITLE
[ui][fix] fix api connector tests

### PR DESCRIPTION
**Pull request by mcada**
//test: @api-connector

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
@mmelko only rest smoke failed, api connectors tests passed, rdy to merge

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/pages/customizations/connectors/wizard/steps/UploadSwaggerSpecification.java [hunk 1]
```diff
@@ -59,7 +59,9 @@ public void upload(String source, String url) {
     }
 
     public void uploadFileFromPath(String path) {
-        $(Input.UPLOAD_A_OPENAPI_FILE).shouldBe(visible).click();
+        // this radio element is always "checked" - due to this bug I have to click two times :(
+        $(Input.UPLOAD_AN_OPENAPI_FILE).shouldBe(visible).click();
+        $(Input.UPLOAD_AN_OPENAPI_FILE).shouldBe(visible).click();
```
Comment (mmelko) [inline ui-tests/src/test/java/io/syndesis/qe/pages/customizations/connectors/wizard/steps/UploadSwaggerSpecification.java]: would be possible to use something like setValue() on radio element?

Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/pages/customizations/connectors/wizard/steps/UploadSwaggerSpecification.java]: Thanks for the idea, setValue() did not work but setSelected() does the job.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmelko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**